common/changes/office-ui-fabric-react/hotfix-AddFocus2Nav-office-ui-fabric-react-5.0_2020-01-22-05-26.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
```

Comment:
The format of this change file looks off - should be like this: https://github.com/OfficeDev/office-ui-fabric-react/pull/10310/files#diff-e60e71efa905e67cc3e13c0fcfca39b0 did you run `yarn change`?

Reply:
This is fine since it's to version 5

Reply:
Actually I don't think a change file is needed in this special hotfix branch because the hotfix process is manual...not totally sure how it works

Reply:
Do I need delete the change file ? or keeping this one is also acceptable ?

Reply:
i see changes in the past have change file : https://github.com/OfficeDev/office-ui-fabric-react/commits/office-ui-fabric-react_v5.79.1.

Reply:
Honestly it isn't needed but it isn't harmful to have it. There is 0 automation for publishing in v5 setup. In v5 era, @dzearing does all hotfix publishing manually.